spring-cloud-gcp-docs/src/main/asciidoc/index.adoc

```diff
@@ -1,8 +1,26 @@
-= Spring Cloud GCP
+= Spring Cloud GCP Reference Documentation
 João André Martins
+:doctype: book
 :toc:
+:toclevels: 4
+:source-highlighter: prettify
+:numbered:
 
-include::overview.adoc[]
+== Introduction
+
+The Spring Cloud GCP project aims at making Google Cloud Platform (GCP) the platform of choice to
+run the popular Spring framework.
```

Comment:
Spring **F**ramework is a brand, not just Spring

Reply:
It's because Spring Framework is its own thing that I didn't want to use it.
I'm afraid that Spring Framework is spring-core, whereas we integrate with Spring as a whole.
Maybe it's still fine to use Spring Framework in that case...

Reply:
might be better to say something like:

```
...aims at bringing the best possible support for Spring applications to the Google Cloud Platform (GCP)
```

(could potentially be explicit that it's actually "Spring Boot" applications, given the reliance on autoconfig, etc)

Overall, it seems that it should be focused on the fact that GCP support will be as strong as possible, without implying that it would be "the platform of choice". Even though I'm personally glad that's the aim ;) , this is a Spring project and shouldn't really include any potentially competitive stance in the doc.

Reply:
How about

`
The Spring Cloud GCP project aims at making the Spring Framework a first-class citizen on the Google Cloud Platform (GCP).
`
?

Reply:
sounds good to me. thanks!